CommandLine.c

```diff
@@ -386,8 +386,10 @@ int CommandLine_run(const char* name, int argc, char** argv) {
    if (flags.pidMatchList)
       Hashtable_delete(flags.pidMatchList);
 
-   /* Delete Settings last, since it can get accessed in the crash handler */
+   /* Delete these last, since they can get accessed in the crash handler */
    Settings_delete(settings);
+   Hashtable_delete(dc);
+   Hashtable_delete(dm);
```

Comment:
Maybe "unregister" those objects from the CRT for shutdown …

Reply:
Like Settings, we want these to hang around until the last pssible minute because the htoprc writing code may be called from segv handler and it needs to be able to map e.g. column identifiers to column names.

Reply:
The SIGSEGV handler should cope with any state of the system, in particular because it can be called in any situation, especially if the system is partly broken. Not properly unwinding here invites UAF bugs and other hardly reproducible states. I can live with some columns not being resolved in the SIGSEGV handler upon shutdown more than getting my system pwn'd by a UAF bug.